frontend/views/containers/payments/PaymentRowTodo.vue

```diff
@@ -86,7 +86,15 @@ export default ({
     }
   },
   methods: {
-    humanDate,
+    humanDateTime (dateTime) {
+      return humanDate(dateTime, {
+        month: 'short',
+        day: 'numeric'
+        // hour: '2-digit',
+        // minute: '2-digit',
+        // hour12: false
+      })
```

Comment:
Could you remove the comment if it's no longer needed and place the options on 1 line?

Reply:
Okay.